.github/workflows/linux-eic-shell.yml

```diff
@@ -871,7 +875,7 @@ jobs:
         - CC: clang
           beam: 18x275
           minq2: 1000
-          detector_config: craterlake
+          detector_config: craterlake_18x275
```

Comment:
If this is the preferred approach I'm happy. Although almost all of the benefits of having the matrix expansion will disappear as this needs to be expanded for the Q2=0 and change the detector config for the 5x41 to `craterlake_5x41` also. Maybe it's not a bad thing to be explicit with the configurations.